### PR TITLE
feat: add helper to create Transfer object

### PR DESCRIPTION
**Pull request by FSM1**
## Description
Implemented a helper to construct a Transfer object.?

## Related Issue Or Context
Closes: #265 

Additionally I have improved the initialization of the SubstrateAssetTransfer class by programatically fetching the Eip712ChainId from the sygmaBridge pallet, instead of requiring users to provide this value manually.

## How Has This Been Tested? Testing details.
Manual testing of new helper in all the examples.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [x] I have updated the documentation locally and in chainbridge-docs.
- [x] I have added tests to cover my changes.
- [x] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: packages/sdk/src/chains/Substrate/assetTransfer.ts [hunk 1]
```diff
@@ -40,19 +41,18 @@ import { SubstrateFee, deposit, getBasicFee } from '.';
...
-export class SubstrateAssetTransfer {
+export class SubstrateAssetTransfer extends BaseAssetTransfer {
   private apiPromise!: ApiPromise;
 
-  public config!: Config;
-
   public async init(
     apiPromise: ApiPromise,
-    parachainId: SubstrateParachain,
     environment: Environment = Environment.LOCAL,
```
Comment (mpetrun5) [inline packages/sdk/src/chains/Substrate/assetTransfer.ts:49]:
```suggestion
    environment: Environment = Environment.MAINNET,
```

File: packages/sdk/src/chains/Substrate/assetTransfer.ts [hunk 2]
```diff
@@ -40,19 +41,18 @@ import { SubstrateFee, deposit, getBasicFee } from '.';
...
   ): Promise<void> {
     this.apiPromise = apiPromise;
+    const parachainId = apiPromise.consts.sygmaBridge.eip712ChainID as U256;
     this.config = new Config();
-    await this.config.init(parachainId.valueOf(), environment);
+    // This is probably not too safe and might overflow
```
Comment (mpetrun5) [inline packages/sdk/src/chains/Substrate/assetTransfer.ts:54]: Overflow?

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

File: packages/sdk/src/chains/Substrate/assetTransfer.ts (hunk 2 above)
Comment (wainola) [reply packages/sdk/src/chains/Substrate/assetTransfer.ts:54]: Does it need to be a number in that case?

Comment (FSM1) [reply packages/sdk/src/chains/Substrate/assetTransfer.ts:54]: could probably get away with a string to be honest

**Review by mpetrun5 — DISMISSED** [review]

**Review by wainola — DISMISSED** [review]

**Review by mpetrun5 — APPROVED** [review]

**Review by MakMuftic — APPROVED** [review]

**Pull request merged**